docs/expiration-emails.md

```diff
@@ -16,10 +16,15 @@ is coming up for renewal. We send the first notice at 20 days before
 your certificate expires, and more notices at 10 days and 1 day before
 it expires.
 
+# When You Get an Expiration Email
+
 If your certificate is already renewed, we won't send an expiry notice. We
 consider a certificate to be renewed if there is a newer certificate
 with the exact same set of names, regardless of which account created it.
-
+If you've issued a new certificate that adds or removes a name relative to your
+old certificate, you will get expiration email about your old certificate. This
+is fine. If you check the certificate currently running on your website, and it
+shows the correct date, you are fine.
```

Comment:
How do you feel about changing "you are fine" to "no action is required and the warning can be ignored"? (Or maybe just "no action is required on your part")